test_mask_upload.py

```
@@ -0,0 +1,148 @@
+def create_test_image():
+    """Create a simple test image"""
+    # Create a 512x512 image with a solid color
+    image = Image.new('RGB', (512, 512), (100, 150, 200))  # Blue-gray color
+
+    # Add some text or pattern
+    from PIL import ImageDraw, ImageFont
+    draw = ImageDraw.Draw(image)
+    try:
+        # Try to use a default font
+        font = ImageFont.load_default()
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)